seahorn/jobs/string_eq_c_str_ignore_case/sea.yaml

```diff
@@ -1,2 +1,3 @@
 verify_options:
   bound: @MAX_STRING_LEN@
+  horn-bmc-tactic: 'smtfd'
```

Comment:
why?

Reply:
With the default solver... and verbose=10, I saw it was going in a loop restarting smt solving mode. With smtfd, we go straight to the sat solver.

It was during refactoring and now i cannot repro the behaviour...reverting change.

Reply:
great. let's be careful and not mix performance and refactoring changes